parser.c

```diff
@@ -126,9 +126,7 @@ static int32_t parse_readbuf_string(struct tinyiiod *iiod, char *str)
 	if (str == ptr || *ptr != '\0' || bytes_count < 0)
```

Comment:
nitpick: You have a typo in the commit message.  number of bytes and functions instead of function.